server/config/config.go

```diff
@@ -144,6 +144,8 @@ type Config struct {
 	Dashboard DashboardConfig `toml:"dashboard" json:"dashboard"`
 
 	ReplicationMode ReplicationModeConfig `toml:"replication-mode" json:"replication-mode"`
+	// EnableRedactLog indicates that whether redact log, 0 is disable. 1 is enable.
+	EnableRedactLog int32 `toml:"enable-redact-log" json:"enable-redact-log"`
```

Comment:
why not use boolean?

Reply:
To keep the same behavior with `tidb` 

https://github.com/pingcap/tidb/blob/fd0299f58ecc174e5ed8f91660299ebac8fa2b9c/config/config.go#L163-L164

Reply:
They use int32 because they need to do atomic operations but we don't. I think we don't have to follow them.